python/ray/experimental/data/read_api.py

```diff
@@ -136,7 +135,6 @@ def remote_read(task: ReadTask) -> Block[T]:
 @PublicAPI(stability="beta")
 def read_parquet(paths: Union[str, List[str]],
                  filesystem: Optional["pyarrow.fs.FileSystem"] = None,
-                 columns: Optional[List[str]] = None,
```

Comment:
Could we keep columns? Since we don't have predicate pushdown this is one of the only ways to get full parquet efficiency in some use cases.

Reply:
Columns never worked. The `columns` arg was never even used in the previous function. https://github.com/ray-project/ray/blob/dad8db46e1ed2acdb8e9b29b1c91b72c2800d60d/python/ray/experimental/data/read_api.py#L137-L218

Reply:
Do we want to consider adding support for `columns` to be within scope of this port?